static/test/javascripts/spec/common/experiments/ab.spec.js

```diff
@@ -297,6 +298,9 @@ define([
             it('should fire the success function when canRun is false', function() {
                 var spy = sinon.spy();
 
+                ab.addTest(test.one);
+                ab.segmentUser();
```

Comment:
Why did we have to add this in? Just makes me slightly worried it might be requires elsewhere due to these changes